custom_components/tuya_local/devices/fisher_summer_air_conditioner.yaml

```diff
@@ -0,0 +1,463 @@
+name: Air conditioner
+products:
+  - id: bf8705744de2dd5ab4ogqj
+    name: Fisher Summer Air Conditioner
+primary_entity:
+  entity: climate
+  icon: "mdi:air-conditioner"
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: false
+          value: "off"
+          icon: "mdi:hvac-off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: auto
+              icon: mdi:thermostat-auto
+              value: heat_cool
+            - dps_val: cold
+              icon: "mdi:snowflake"
+              value: cool
+            - dps_val: wind
+              icon: "mdi:fan"
+              value: fan_only
+            - dps_val: wet
+              icon: "mdi:water"
+              value: dry
+            - dps_val: hot
+              value: heat
+              icon: "mdi:heat-wave"
+    - id: 2
+      name: temperature
+      type: integer
+      range:
+        min: 160
+        max: 310
+      mapping:
+        - scale: 10  
+          step: 5 
+      unit: C
+    - id: 3
+      name: current_temperature
+      type: integer
+      readonly: true
+    - id: 4
+      name: mode
+      type: string
+    - id: 5
+      name: fan_mode
+      type: string
+      mapping:
+        - dps_val: auto
+          value: Auto
```

Comment:
These should use the translations from https://github.com/home-assistant/core/blob/dev/homeassistant/components/climate/strings.json#L40 if necessary supplemented by `translation_key: aircon_extra` values from https://github.com/make-all/tuya-local/blob/main/custom_components/tuya_local/translations/en.json#L65 - the value to use here is the key (left hand side) in the strings files, and is case-sensitive so should be all lower case (lower case is used to easily differentiate where translations are used in the config values)